inc/src/phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
```

Comment:
I'm not sure if this is the best place for this config file. The idea behind `inc/src/` is to hold source code. All of our other config (`phpcs`, `phpunit`, etc.) is in the root `/` so it may make more sense too live there.